packages/astro/src/actions/runtime/middleware.ts

```diff
@@ -99,7 +99,8 @@ async function handlePost({
 	if (contentType && hasContentType(contentType, formContentTypes)) {
 		formData = await request.clone().formData();
 	}
-	const action = baseAction.bind(context);
+	const { getActionResult, callAction, props, redirect, ...actionAPIContext } = context;
+	const action = baseAction.bind(actionAPIContext);
 	const actionResult = await action(formData);
```

Comment:
Is this a typo?

Reply:
@ematipico I am not sure what you are referring too. I double-checked everything and couldn't find any typo.

Reply:
Yeah, sorry, from mobile it was like you added this line. carry on!